src/main/java/me/xemor/configurationdata/entity/EntityData.java

```diff
@@ -86,7 +87,11 @@ private EntityData(EntityType entityType) {
     }
 
     public Entity spawnEntity(@NotNull Location location) {
-        Entity entity = location.getWorld().spawnEntity(location, entityType);
+        return spawnEntity(location.getWorld(), location);
+    }
```

Comment:
Why did you re add this method?

Reply:
> Why did you re add this method?

It was being used throughout SkillsLibrary and it made me realise that it's probably safer to add this back. Equally happy to remove this again and just update in SkillsLibrary

Reply:
I think I might have ported over myself, do a pull on your skillslibrary

Reply:
Ahh, my branch is out of date